Source/Scene/ClassificationModel.js

```diff
@@ -124,7 +130,10 @@ define([
 
         if (gltf instanceof Uint8Array) {
             // Binary glTF
-            gltf = parseBinaryGltf(gltf);
+            gltf = parseBinaryGltf(gltf); // Updates to 2.0 and adds pipeline extras
+            addDefaults(gltf);
+            processModelMaterialsCommon(gltf);
+            processPbrMetallicRoughness(gltf);
```

Comment:
Added these lines because other code in `ClassificationModel` expected `gltf.techniques` to exist.

@ggetz you may need to add these changes to https://github.com/AnalyticalGraphicsInc/cesium/pull/6805 as well.

Reply:
Noted in that PR.